### PR TITLE
Fix html validation errors (fixes #44)

### LINKED ISSUE #44: HTML validation errors
`index.html` and `about/index.html` contain a number of HTML validation errors, some of which affect accessibility.

As reported by validator.nu:

```
"file:index.html":10.5-10.105: error: Bad value “https://fonts.googleapis.com/css?family=Crimson+Text|Open+Sans:400,700” for attribute “href” on element “link”: Illegal character in query: “|” is not allowed.
"file:index.html":47.15-47.166: error: The “frameborder” attribute on the “iframe” element is obsolete. Use CSS instead.
"file:index.html":56.9-56.14: error: End tag “div” seen, but there were open elements.
"file:index.html":43.9-43.35: error: Unclosed element “section”.
"file:index.html":120.47-120.52: error: Named character reference was not terminated by a semicolon. (Or “&” should have been escaped as “&amp;”.)
"file:index.html":120.107-120.112: error: Named character reference was not terminated by a semicolon. (Or “&” should have been escaped as “&amp;”.)
"file:index.html":162.17-162.72: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:index.html":175.17-175.74: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:index.html":188.17-188.76: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:index.html":203.17-203.70: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:index.html":259.5-259.10: error: Stray end tag “div”.
"file:about/index.html":10.5-10.105: error: Bad value “https://fonts.googleapis.com/css?family=Crimson+Text|Open+Sans:400,700” for attribute “href” on element “link”: Illegal character in query: “|” is not allowed.
"file:about/index.html":62.17-62.70: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":104.17-104.76: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":134.83-134.88: error: Named character reference was not terminated by a semicolon. (Or “&” should have been escaped as “&amp;”.)
"file:about/index.html":134.89-134.93: error: Named character reference was not terminated by a semicolon. (Or “&” should have been escaped as “&amp;”.)
"file:about/index.html":135.77-135.82: error: Named character reference was not terminated by a semicolon. (Or “&” should have been escaped as “&amp;”.)
"file:about/index.html":135.83-135.87: error: Named character reference was not terminated by a semicolon. (Or “&” should have been escaped as “&amp;”.)
"file:about/index.html":136.71-137.21: error: Saw “<” when expecting an attribute name. Probable cause: Missing “>” immediately before.
"file:about/index.html":136.71-137.23: error: A slash was not immediately followed by “>”.
"file:about/index.html":136.71-137.24: error: End tag had attributes.
"file:about/index.html":145.19-145.69: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":184.19-184.72: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":193.25-193.28: error: Element “li” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)
"file:about/index.html":196.25-196.28: error: Element “li” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)
"file:about/index.html":199.25-199.28: error: Element “li” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)
"file:about/index.html":202.25-202.28: error: Element “li” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)
"file:about/index.html":209.23-209.26: error: No “p” element in scope but a “p” end tag seen.
"file:about/index.html":218.19-218.68: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":255.19-255.70: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":275.19-275.69: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":360.19-360.74: error: An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
"file:about/index.html":375.9-375.14: error: End tag “div” seen, but there were open elements.
"file:about/index.html":35.7-35.58: error: Unclosed element “section”.
"file:about/index.html":376.7-376.16: error: Stray end tag “section”.
"file:about/index.html":395.5-395.10: error: Stray end tag “div”.
```

### PR DESCRIPTION
**Pull request by adamobeng**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -131,9 +131,9 @@ <h3 class='card-title'>Chad Elish</h3>
                         Apple IT consultant for a small firm.
                       </li>
                     </ul>
-                    <p><a href="http://www.linkedin.com/in/chadelish">LinkedIn</a>&nbsp&nbsp
-                      <a href="http://www.twitter.com/chadisrad">Twitter</a>&nbsp&nbsp
-                      <a href="http://facebook.com/israd">Facebook</a></p
+                    <p><a href="http://www.linkedin.com/in/chadelish">LinkedIn</a>&nbsp;&nbsp;
```
Comment (danawoodman) [inline about/index.html]: Could you actually replace these with spaces instead? I think it is a copy/paste issue

File: about/index.html [hunk 2]
```diff
@@ -142,7 +142,7 @@ <h3 class='card-title'>Chad Elish</h3>
             <div class='row'>
               <div class='col-lg-6'>
                 <div class='card'>
-                  <img class='card-img-top' src='../images/dana.jpg'>
+                  <img class='card-img-top' src='../images/dana.jpg' alt='Profile image of Dana Woodman (and cat)'>
```
Comment (danawoodman) [inline about/index.html]: "(and cat, Boots)", she needs due credit! 😝

File: index.html [hunk 3]
```diff
@@ -117,7 +117,7 @@ <h1 class='display-3'>Nation of Makers</h1>
                 collaboration fosters a culture of abundance.
               </p>
               <p>
-                Below are some <i>examples</i>&nbsp of the initiatives that this organization <i>could</i>&nbsp lead. These activities correspond to key needs
+                Below are some <i>examples</i>&nbsp; of the initiatives that this organization <i>could</i>&nbsp; lead. These activities correspond to key needs
```
Comment (danawoodman) [inline index.html]: Could you replace with normal spaces?

**Review by danawoodman — CHANGES_REQUESTED** [review]
Awesome work and thank you, just a few small tweaks and I'll merge!

**Comment by adamobeng** [comment]
@danawoodman Have done!

**Review by danawoodman — APPROVED** [review]

**Comment by danawoodman** [comment]
@adamobeng thanks so much! 🍻

**Pull request merged**